modin/conftest.py

```diff
@@ -319,11 +360,12 @@ def pytest_configure(config):
     if execution is None:
         return
 
-    if execution == BASE_EXECUTION_NAME:
-        set_base_execution(BASE_EXECUTION_NAME)
-        config.addinivalue_line(
-            "filterwarnings", "default:.*defaulting to pandas.*:UserWarning"
-        )
+    if execution == "BaseOnPython":
+        set_base_on_python_execution()
+        config.addinivalue_line("filterwarnings", default_to_pandas_ignore_string)
+    elif execution == "Client":
+        config.addinivalue_line("filterwarnings", default_to_pandas_ignore_string)
```

Comment:
Is it necessary to reverse the order of lines (367, 368) compared to (364, 365)?

Reply:
Nope, I made them match.